### ISSUE TITLE
Altering Encoding Parameter - MaxAudioBitrate and MaxVideoBitrate do not work

### ISSUE THREAD

**Issue by anooshcnayak** [twilio/video-quickstart-android#591]
### Description
Altering MaxAudioBitrate and MaxVideoBitrate do not have any significance. **P2P** Room - **2** Participants & also **Group** - 2 Participants

### Settings:
MaxAudioBitrate - **1**, MaxVideoBitrate - **1**

Resolution & FPS are set 5 x 5 @ 1 FPS. Simulcast is **OFF**

#### Code

```java
new ConnectOptions.Builder(this.accessToken).encodingParameters(new EncodingParameters(1, 1));
```

#### Expected Behavior

Data consumption to be minimal, Network Utilization should be 2 Kilobits per second as configured.

#### Actual Behavior

Average Network Utilization seems to be around **80** Kilobits per second. Data consumption calculation w.r.t 2 Kilobits for the duration measured does not add up

**My Understanding:**
Whatever resolution is captured, based on bitrate, it downscales the video (fps+ resolution) to not exceed the bitrate threshold. Would like to know what is happening here?

#### Reproduces how Often

Everytime

### Versions

Latest Version. (6.0)

#### Video Android SDK

Android 9

#### Android API

29

#### Android Device

Oneplus 6T

**Comment by aaalaniz** [comment]
Hey @anooshcnayak 

Can you provide a Room SID so we can investigate further?

Thanks!

**Comment by anooshcnayak** [comment]
@aaalaniz 

**Two Tests:** Duration: 10 Minutes
1. P2P Room - 2 Participants - Data Usage: ~ 10 MB - Audio & Video Bitrate - 1Kbps (kilobits) - Min & Max Video Dimensions: 30 x 30
    Room SID - RM7ba54b3745689f1324e89e9e73141cd1
2. Group Room - 2 Participants - Data Usage - ~ 10 MB -  Audio & Video Bitrate - 1Kbps (kilobits) - Min & Max Video Dimensions: 30 x 30
    Room SID - RM59f7d7218d4be8a9008dd060d7a4dc2d 
    Note: If you check the above room - its duration might be 20 mins, but the data usage was checked on the device with the participant PA465922d4b31e7eb6860a013f6a795796 which was ~ 13 minutes

Also, Simulcast is **OFF**.

**Comment by anooshcnayak** [comment]
Hey @aaalaniz  Any update on this?

**Comment by afalls-twilio** [comment]
@anooshcnayak Sorry for the late response, this is happening because 1 bps is not a valid value and so it gets ignored by WebRTC.

**Comment by afalls-twilio** [comment]
@anooshcnayak Sorry for the long delay. The bitrate code has been significantly updated and those changes will be available in a future release, so please stay tuned.. (unfortunately, not the upcoming one)

**Comment by afalls-twilio** [comment]
Max bitrate using Plan-B was fixed in 7.3.0
